### ISSUE TITLE
Cant log in with Dreame L20 Ultra

### ISSUE THREAD

**Issue by pillemats** [Tasshack/dreame-vacuum#536]
hey there,

i want to use the plugin with version 1.04 (Before i used the beta but it dont works many days ago)
so i deleted the plugin an reinstall.

but now i cant cant login 

<img width="539" alt="Bildschirmfoto 2024-05-10 um 07 40 19" src="https://github.com/Tasshack/dreame-vacuum/assets/26164977/fc004613-1c36-4980-89a0-3e6ed8bf063f">

what can i do?

HA and plugin are the newest versions

**Comment by pillemats** [comment]
I tried this but it don’t work

**Comment by pillemats** [comment]
Yes is changed both… i willl try tomorrow and give you a feedback

**Comment by pillemats** [comment]
yes, i tried. i reboot my router and also tried a 4g router. Same result


but another message:
<img width="567" alt="Bildschirmfoto 2024-05-10 um 14 31 23" src="https://github.com/Tasshack/dreame-vacuum/assets/26164977/a251e98d-feb0-40a6-a445-5b4a375d1a8c">

**Comment by pillemats** [comment]
i tried this many times, but i cant install any beta version! it shown this:

<img width="543" alt="Bildschirmfoto 2024-05-10 um 16 44 36" src="https://github.com/Tasshack/dreame-vacuum/assets/26164977/4f51245a-2fd8-4360-8b13-9198d58a4d5b">

if i choose b14 it automaticly switches back to 1.0.4

**Comment by pillemats** [comment]
sorry, but i tried this also! It switched back to stock 1.0.4!

**Comment by AndreaCCIE** [comment]
> sorry, but i tried this also! It switched back to stock 1.0.4! this is a bug i think!
> 
I have the same issue: even if I install the b14, after HA reload it shows 1.0.4
